ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/sum_with_out_max_min.rb

```diff
@@ -0,0 +1,18 @@
+
+class SumWithOutMaxMin
+    
+    def self.sort_array_of_intergers(array_of_intergers)
+        array_of_intergers.sort
+    end
+
+    def self.eliminate_min_and_max(array_of_intergers)
```

Comment:
This is mostly nitpicking, but I'd argue that `remove_min_and_max` makes more sense in the context of what you're doing to the array.